app/main/forms/frameworks.py

```diff
@@ -34,3 +34,17 @@ class AcceptAgreementVariationForm(Form):
             DataRequired(message="You can only save and continue if you agree to the proposed changes.")
         ]
     )
+
+
+class ReuseDeclarationForm(Form):
+    """Form for the reuse declaration page. One yes no question.
+
+    `reuse` is a yes no whether they want to reuse a framework.
+    `old_framework` is a hidden field allowing us to pass back the framework slug of the framework they are choosing to
+    reuse.
+    """
+    reuse = BooleanField(
+        'Do you want to reuse the answers from your earlier declaration?',
+        validators=[InputRequired()]
```

Comment:
For future reference, that default message could be something like ‘You must answer this question.' which is less technical and more user-friendly. But changing the default is a bigger piece of work, I guess, so I'm not suggesting we make this change now.